app.json

```diff
@@ -0,0 +1,30 @@
+{
+  "expo": {
+    "name": "GPSPosition",
+    "slug": "GPSProject",
+    "privacy": "public",
+    "sdkVersion": "35.0.0",
+    "platforms": [
+      "ios",
+      "android",
+      "web"
+    ],
+    "version": "1.0.0",
+    "orientation": "portrait",
+    "icon": "./assets/icon.png",
+    "splash": {
+      "image": "./assets/splash.png",
```

Comment:
While it runs, we'll get an error saying that this is not there.

Maybe we can get a quick icon and splash done?